nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/JoltTransformJSON.java

```diff
@@ -95,6 +99,24 @@
             .required(false)
             .build();
 
+    public static final PropertyDescriptor CUSTOM_CLASS = new PropertyDescriptor.Builder()
+            .name("jolt-custom-class")
+            .displayName("Custom Transformation Class Name")
+            .description("Fully Qualified Class Name for Custom Transformation  Module Directory should be specified")
```

Comment:
I just took that second part out and will elaborate in the additionalDetails page and a bit on the custom module directory description.